databook/anatomy/microns-em/em-querying-tables.md

```diff
@@ -127,7 +130,7 @@ We can get the whole table as a DataFrame:
 ```{code-cell}
 :tags: [remove-stderr]
 
-cell_type_df = client.materialize.tables.aibs_soma_nuc_metamodel_preds_v117().query()
+cell_type_df = client.materialize.tables.aibs_metamodel_celltypes_v661().query()
```

Comment:
Right above this in the markdown we say we're accessing `aibs_soma_nuc_metamodel_preds_v117`, can we update this text to be consistent with the code?

Reply:
Yes! Good catch.